environmental-values.csl

```diff
@@ -13,22 +13,25 @@
     <category citation-format="author-date"/>
     <category field="humanities"/>
     <issn>0963-2719</issn>
-    <updated>2017-07-24T09:28:04+00:00</updated>
+    <updated>2021-03-28T08:30:02+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <macro name="title">
     <choose>
       <if type="bill book graphic legal_case motion_picture report song" match="any">
         <text variable="title" text-case="capitalize-first" font-style="italic"/>
       </if>
+      <else-if type="article-journal" match="any">
+        <text variable="title" text-case="capitalize-first" prefix="'" suffix="'"/>
```

Comment:
set quotes using `quotes="true"`

Reply:
I was using the visual editor and the only quotes I could get were double quotes "" while the style asks for single quotes ''. How does one add those?

Reply:
`quotes="true"` sets quotation marks based on the locale. Since the style is British English (`en-GB`), they'll be single quotation marks.